.gitignore

```diff
@@ -1,7 +1,8 @@
 *-compiled*
 node_modules
 jspm_packages
-.idea
+.idea/
```

Comment:
I'm puzzled by this

Reply:
Hmm... I think :sweat_smile: I have wanted to say that our .gitignore file should have more strict definition:

> If you put `foo`, it will match all files and directories named `foo`
> If you put `foo/`, it will only match directories named `foo`.

https://stackoverflow.com/a/23857256

Reply:
Oh! I see, okay.